content/docs/components-and-props.md

```diff
@@ -42,27 +42,27 @@ class Welcome extends React.Component {
 }
 ```
 
-The above two components are equivalent from React's point of view.
+上述兩種元件的在 React 中是同等的。
 
-Classes have some additional features that we will discuss in the [next sections](/docs/state-and-lifecycle.html). Until then, we will use function components for their conciseness.
+我們將會在[下一個章節]((/docs/state-and-lifecycle.html))探討 class 所擁有的額外特性，但在那之前，我們會使用函式元件來保持簡潔。
 
-## Rendering a Component {#rendering-a-component}
+## 描繪一個元件 {#rendering-a-component}
```

Comment:
I know at least a few reviewers want to keep `render` in English to avoid confusion.